.github/workflows/php-code-analysis.yml

```diff
@@ -124,20 +123,14 @@ jobs:
                     fi
 
                     if [ "$(make -p | grep 'phpstan:')" = "phpstan:" ]; then
-                        echo "phpstan_exists=true" >> $GITHUB_OUTPUT
+                        cd /shopware
+                        php bin/console -e production -q # Initialise DIC (var/cache/production____REVISION____) for phpstan
```

Comment:
couldn't you execute that first? instead of jumping into the plugin, then back to shopware, then back to the plugin. 

so in the end, you only jump once to the plugin directory

Reply:
You are right.
Done that